### PR TITLE
add simple button GUI to nxt-screen.py

### PR DESCRIPTION
**Pull request by arohrbacher**
add simple button GUI to nxt-screen.py to emulate the physical NXT buttons; this allows remote controlling the NXT brick

### CODE REVIEW

**Comment by arohrbacher** [comment]
also see this issue:
https://github.com/schodet/nxt-python/issues/195

File: examples/nxt-screen.py [hunk 1]
```diff
@@ -12,12 +14,16 @@
 this code basically reimplements what
 https://bricxcc.sourceforge.net/utilities.html -> "NeXTScreen"
 in a less obscure programming language ;-)
+
+arohrbacher, 2.1.2024: Added code to provide a GUI representing the NXT physical buttons so that the NXT 
+brick can be remotely controlled
+
```
Comment (schodet) [inline examples/nxt-screen.py:20]: This does not need to be in source file, it is provided in history.

File: examples/nxt-screen.py [hunk 2]
```diff
@@ -73,6 +86,7 @@ def main_loop():
     b.play_tone(440, 250)
 
     while True:
+        mouse = pygame.mouse.get_pos()
```
Comment (schodet) [inline examples/nxt-screen.py:89]: You can use event.pos instead and get accurate mouse position at the time of the click.

**Review by schodet — CHANGES_REQUESTED** [review]

**Comment by schodet** [comment]
Thanks for the merge request, it works nicely. Could you address the two comments and squash the changes into one commit?

Thanks.

**Comment by arohrbacher** [comment]
Thanks for the comments; I adressed your comments: removed my explanation at the beginning of the file and used event.pos to get the mouse coordinates. Now I have an other commit...
However, I don't quite know how to squash the three commits on nxt-screen.py - apologies for that, I am pretty new to git/github and version control...

File: examples/nxt-screen.py (hunk 1 above)
Comment (arohrbacher) [reply examples/nxt-screen.py]: removed lines 18 and 19

File: examples/nxt-screen.py (hunk 2 above)
Comment (arohrbacher) [reply examples/nxt-screen.py]: removed line 89 and used event.pos

**Comment by schodet** [comment]
OK, I did the squash and pushed to master, thanks for your change!

**Pull request closed without merging**